lib/libyear_bundler/cli.rb

```diff
@@ -60,39 +53,32 @@ def query
     end
 
     def report
-      @_report ||= Report.new(query, @argv)
+      @_report ||= Report.new(query, @options)
     end
 
     def unexpected_options
       @_unexpected_options ||= begin
-        options = @argv.select { |arg| arg.start_with?("--") }
+        options = @options.select { |arg| arg.start_with?("--") }
         options.each_with_object([]) do |arg, memo|
           memo << arg unless OPTIONS.include?(arg)
         end
       end
     end
```

Comment:
It seems `unexpected_options` is no longer used?

Reply:
Yes